docs/source/api.rst

```diff
@@ -257,6 +549,18 @@ Lifetime
 .. literalinclude:: schemas/request/Lifetime.json5
    :language: javascript
 
+=================================  =======  ==================  ==============  ========  ==================  ============================================== 
+Property                           Type     Units               Constraints     Required  Default             Notes
+=================================  =======  ==================  ==============  ========  ==================  ==============================================       
+``replacementCost``                float    ``units.monetary``  >=0             see [#]_                      Default values not supported currently
+``endOfLifeDate``                  date                         in the future   see [*]_      
+``effectiveUsefulLifeDays``        integer  days                >0              see [*]_   BSA    
+``installedDate``                  date                         in the past     see [*]_   PSC, BSA
+=================================  =======  ==================  ==============  ========  ==================  ============================================== 
+
+.. [#] Required to run status quo timeline.
+.. [*] Two of these three properties (``endOfLifeDate``, ``effectiveUsefulLifeDays``, ``installedDate``) are required to be included in the status quo timeline.
```

Comment:
I think footnotes got messed at this point.

![image](https://github.com/RadiantLabs/modeling-api-docs/assets/10285345/b7b8bb73-d422-4149-b637-2ed9cf875efd)

I think we can have named footnotes, so you can repeat the same reference and point to one single footnote. Although, we'll need to take a look at the RTD docs.